pkg/project/resource_project_repo.go

```diff
@@ -127,10 +127,16 @@ var updateRepos = func(projectKey string, terraformRepoKeys []RepoKey, m interfa
 	return readRepos(projectKey, m)
 }
 
+var retry5xxRange = func(response *resty.Response, err error) bool {
+	retryableHttpStatusCode := []int{408, 500, 502, 503, 504}
```

Comment:
Please use the constants from `http` package. https://pkg.go.dev/net/http#pkg-constants

Reply:
I think you can, and should do this with a `case` statement instead of a function, because you can have multiple matches to a single case

https://www.geeksforgeeks.org/golang-program-that-uses-switch-multiple-value-cases/

And yes, use `http.` status constants